Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: help build up down logs clean ps
+.PHONY: help build build-server run start check-deps setup-certs install
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)